generic_expression_patterns_modules/download_recount2_data.R

```diff
@@ -123,9 +119,62 @@ base_dir){
   data_counts_all <- t(data_counts_all)
 
   ## Save counts matrix to file
-  write.table(data_counts_all,
-            paste(local_dir,'/recount2_compendium_data.tsv',sep=""),
-            sep='\t',
-            row.names=TRUE,
-            col.names=NA)
-}
\ No newline at end of file
+  write.table(
+    data_counts_all,
+    paste(local_dir,'/recount2_compendium_data.tsv',sep = ""),
+    sep = '\t',
+    row.names = TRUE,
+    col.names = NA
+  )
+}
+
+# This function downloads all recount2 `rse_gene.Rdata` files and generates a
+# transposed data count file (`t_data_counts.tsv`) based on each `rse_gene.Rdata`.
+download__all_recount2 <- function (metadata_dir, download_dir) {
```

Comment:
Do you recall how long the download took? I wonder if it'd be helpful to add in a comment so that the person knows how long they can expect to wait

Reply:
I thought of adding some comments on how long the downloading process may take, but it largely depends on the user's network speed (as I mentioned in one of my earlier comments). Maybe I should add a comment that sounds vague, such as `The downloading process may take 30 minutes to a few hours, depending on your network speed.`

Reply:
I see, in that case I'll defer to you. I am not as expert here, so I just assumed the download would be a static amount of time, which in that case I think the time estimate would be helpful but perhaps not given what you're saying